### PR TITLE
Add deposit/withdrawal wstETH test variants for different state of L1/L2

### PR DESCRIPTION
**Pull request by kovalgek**
Use old data format for deposits to be able to test using old L1 bridge state.

### CODE REVIEW

File: test/integration/_bridging-non-rebasable.ts [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+
+      await assert.emits(l1CrossDomainMessenger, tx, "SentMessage", [
+        l2ERC20ExtendedTokensBridge.address,
+        l1LidoTokensBridge.address,
+        l2DepositCalldata,
+        messageNonce,
+        200_000,
+      ]);
+
+      assert.equalBN(
```
Comment (TheDZhon) [inline test/integration/_bridging-non-rebasable.ts:88]: Eh, big numbers

File: test/integration/_bridging-non-rebasable.ts [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+}
+
+export function ctxFactory(
+  useNewDataFormatOnL1: boolean,
+  useNewDataFormatOnL2: boolean,
+  depositAmount: BigNumber,
+  withdrawalAmount: BigNumber
+) {
+  return async () => {
+    const networkName = env.network("TESTING_OPT_NETWORK", "mainnet");
```
Comment (TheDZhon) [inline test/integration/_bridging-non-rebasable.ts:546]: Looks weird to have testing and mainnet together tbh

File: test/integration/bridging-non-rebasable-new_L1-old_L2.integration.test.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { BigNumber } from 'ethers'
+import { wei } from "../../utils/wei";
+import { scenario } from "../../utils/testing";
+import { bridgingTestsSuit, ctxFactory } from "./_bridging-non-rebasable";
+
+bridgingTestsSuit(
+  scenario(
+    "Optimism :: Bridging X non-rebasable token integration test",
+    ctxFactory(
+      true,
```
Comment (TheDZhon) [inline test/integration/bridging-non-rebasable-new_L1-old_L2.integration.test.ts]: Can we make these constants more readable either introducing aliases or least with some comments? 💪

**Review by TheDZhon — APPROVED** [review]
Love it 🙏

**Comment by TheDZhon** [comment]
Only one rather dumb question: why do we test only non rebasable tokens here?

File: test/integration/_bridging-non-rebasable.ts (hunk 2 above)
Comment (kovalgek) [reply test/integration/_bridging-non-rebasable.ts:546]: It's related to the network name only. Integration tests are running only on forks.

**Review by arwer13 — APPROVED** [review]

**Comment by kovalgek** [comment]
> Only one rather dumb question: why do we test only non rebasable tokens here?

Added readme file to tests folder.

**Pull request merged**